deployer/README.md

```diff
@@ -17,172 +17,191 @@ pip install --editable .
-│ generate-helm-upgrade-jobs          Analyze added or modified files from a       │
-│                                     GitHub Pull Request and decide which         │
-│                                     clusters and/or hubs require helm upgrades   │
-│                                     to be performed for their *hub helm charts   │
-│                                     or the support helm chart.                   │
-│ new-grafana-token                   Generate an API token for the cluster's      │
-│                                     Grafana `deployer` service account and store │
-│                                     it encrypted inside a                        │
-│                                     `enc-grafana-token.secret.yaml` file.        │
-│ run-hub-health-check                Run a health check on a given hub on a given │
-│                                     cluster. Optionally check scaling of dask    │
-│                                     workers if the hub is a daskhub.             │
-│ start-docker-proxy                  Proxy a docker daemon from a remote cluster  │
-│                                     to local port 23760.                         │
-│ update-central-grafana-datasources  Update the central grafana with datasources  │
-│                                     for all clusters prometheus instances        │
-│ use-cluster-credentials             Pop a new shell or execute a command after   │
-│                                     authenticating to the given cluster using    │
-│                                     the deployer's credentials                   │
-│ user-logs                           Display logs from the notebook pod of a      │
-│                                     given user                                   │
-│ validate                            Validate cluster.yaml and non-encrypted helm │
-│                                     config for given hub                         │
-╰──────────────────────────────────────────────────────────────────────────────────╯
 
+### The `utils` directory
+This is where utility functions are stored. They are to be imported and used throughout the entire codebase of the deployer.
+
+```bash
+└── utils
+    ├── file_acquisition.py
+    └── rendering.py
 ```
 
-### `deploy`
+### The `commands` directory
 
-This function is used to deploy changes to a hub (or list of hubs), or install it if it does not yet exist.
-It takes a name of a cluster and a name of a hub (or list of names) as arguments, gathers together the config files under `/config/clusters` that describe the individual hub(s), and runs `helm upgrade` with these files passed as `--values` arguments.
+This is the directory where all of the code related to the main `deployer` sub-commands is stored.
 
-**Command line usage:**
+Each sub-commands's functions are stored:
+- either in a single Python file that ends in `_cmd` or `_commands`
```

Comment:
Given we're already in a subdirectory called `commands`, I think it's ok to not have this additional suffix. Less clutter that way.